contracts/hts-precompile/ERC20Contract.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+contract ERC20Contract {
```

Comment:
Looks good.
Please wait on https://github.com/hashgraph/hedera-smart-contracts/pull/57 to go in and then add test coverage for at least the `name`, `symbol`, `decimals` and `totalSupply` methods.

Reminder the goal would be the initial coverage we added in the relay would not be needed and this repo would be the source of basic and simple contracts with their tests using hardhat and local-node.
Nothing complex, just easy to view and run from a pure web3 (less Hedera) point of view